.github/workflows/build-linux-arm.yml

```
@@ -0,0 +1,169 @@
+          mkdir -p patches libtor-src/patches
+          touch patches/.keep libtor-src/patches/.keep
+          cargo build --release --verbose --target ${{ matrix.target }} --features vendored-openssl
+
+      - name: Build eltord (Linux cross-compilation)
+        if: matrix.setup-script == 'setup-linux'
+        run: |
+          cd $BUILD_DIR/eltord
+          cargo build --release --verbose --target ${{ matrix.target }} --features vendored-openssl
+
+      - name: Copy artifacts
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:4: Double quote to prevent globbing and word splitting

(shellcheck)